README.md

```diff
@@ -1,16 +1,42 @@
-# mt32.net
-
-[![Netlify Status](https://api.netlify.com/api/v1/badges/4e30a997-7aa8-4fa2-8856-93e9b9a5ee39/deploy-status)](https://app.netlify.com/sites/mtorials-test/deploys)
-
-A hugo website deployed with netlify!
-
-## License
-
-### Theme
-
-The theme is licensed under the MIT license. Feel free to use and modify it for your own projects.
-
-### Content
-
-The content of this website is licesed under the [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.
-Content means the actual posts and texts under the `content` directory.
+<img src="https://user-images.githubusercontent.com/39274150/111981613-30d1ff00-8b08-11eb-8767-b89f2e0970be.png"/>
+<h1>https://mt32.net</h1>
+<h2>The blog for all computer interested</h2>
+
+ Netlify  
+|----|
+[![Netlify Status](https://api.netlify.com/api/v1/badges/4e30a997-7aa8-4fa2-8856-93e9b9a5ee39/deploy-status)](https://app.netlify.com/sites/mtorials-test/deploys)
+
+# About mt32.net
+
+`mt32.net` is running by @mtorials and @universum two hobby programmer. 
```

Comment:
I am the only one working on this....

Reply:
i was use the source on your about page. You should update this site.

Reply:
I was writen from your about source. https://mt32.net/about. I will remove universum.